C/kmsg_log.c

```diff
@@ -5,52 +5,55 @@
 #include <fcntl.h>
 #include <string.h>
 
+#define ALT_LOG "/sdcard/lible.log"
+
 int klog(__attribute__((unused)) int level,__attribute__((unused)) char * a )
 {
+
+ char *buf = malloc(sizeof(char) * 256);
```

Comment:
We can't initialize with 'buf' as it is inexistent at this time. 
However, when 'a' will be used, we will be able to use something like this: 
malloc(sizeof(char) * sizeof(a));